proco/connection_statistics/api.py

```diff
@@ -24,3 +31,28 @@ def get(self, request, *args, **kwargs):
             'countries_joined': countries_joined,
         }
         return Response(data=data)
+
+
+class CountryWeekStatsAPIView(APIView):
+    permission_classes = (AllowAny,)
+
+    def get(self, request, country_id, week, year, *args, **kwargs):
+        stat_instance = get_object_or_404(CountryWeeklyStatus, country_id=country_id, week=week, year=year)
```

Comment:
it would be better to use builtin functionality as much as possible. please move this code to get_object and set serializer_class=CountryWeeklyStatusSerializer